src/main/java/com/example/library/repository/BookRepositoryImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.example.library.repository;
+
+import com.example.library.exception.CapacityException;
+import com.example.library.exception.NoBookException;
+import com.example.library.model.Book;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.util.HashMap;
+
+@Repository("bookRepository")
+public class BookRepositoryImpl implements BookRepository {
+    private HashMap<Integer, Book> books;
+
+    @Value("${capacity}")
```

Comment:
@value też można w konstruktor wbić potem lepiej się to ustawia w testach

Reply:
Ogólnei to capacity ustaw sobie lepiej w serwisie. Później jak przejdziesz na normalne spring repository to tam logiki biznesowej i tak nie ustawisz

Reply:
W sensie, że ustawiać capacity w serwisie, czy całe capacity wyciagnac z repository i trzymac w serwise?

Reply:
Całe trzymać w serwisie. Dla tego że to logika biznesowa a nie persystancji. Więc to w serwisie powinno być.